src/source/reference/get-metrics-ads.rst

```diff
@@ -4,69 +4,74 @@
+
 
 Report Columns
 ~~~~~~~~~~~~~~
 
-Both excel and csv contain the following columns: (TBW)
-
-===================      ==========================================================================
-Name                     Description
-===================      ==========================================================================
-Date (Aggregated)        Date of the report row, grouped hourly, daily, weekly, monthly, or yearly
-Ad id                    Id of the ad
-L1 Category              Level 1 category name
-L2 Category              Level 2 category name
-L3 Category              Level 3 category name if it exists
-Title                    Ad title
-Start Date               The creation date of the ad
-End Date                 If the ad is deleted, deletion date of the ad
-Cpc                      Cpc of the ad for which performance metrics are calculated
-Total spent              Total amount spent for this ad
-Clicks                   Number of clicks that the ad received
-Impressions              Number of impressions that the ad received
-CTR                      Click through rate in %
-URL clicks               Number of URL clicks that the ad received
-Emails                   Number of email events that the ad received
-Engagement CTR           Engagement conversion rate in %. Currently = (URL clicks + Emails)/Clicks
-Region                   Region of the ad
-Vendor id                Vendor id of the ad
-===================      ==========================================================================
+Both the excel and the csv formats contain the following columns by default:
+
+===================   =================   ===============================================================
+Name                   Field               Description
+===================   =================   ===============================================================
+Date (Aggregated)     ``date``             Date of the report row, grouped daily, weekly, monthly, or yearly. For daily and weekly aggregation the format is ``YYYY-MM-DD``, for monthly aggregation - ``YYYY-MM``, and for yearly - ``YYYY``. All dates are in tenant timezone.
+Ad ID                 ``adID``             ID of the ad
+L1 Category           ``L1Category``       Level 1 category name
+L2 Category           ``L2Category``       Level 2 category name
+L3 Category           ``L3Category``       Level 3 category name, if applicable
+Title                 ``title``            Title of the ad
+Start Date            ``startDate``        Creation date of the ad
+End Date              ``endDate``          If the ad is deleted, deletion date of the ad
+CPC                   ``CPC``              CPC of the ad for which performance metrics are calculated, in Euros
```

Comment:
We should change "Euros" into "Local Currency"

Reply:
done